plugins/modules/inventory_intent.py

```diff
@@ -1238,25 +1243,25 @@ def resync_devices(self):
                             self.msg = "Device Resynced get failed because of {0}".format(failure_reason)
                         else:
                             self.msg = "Device Resynced get failed."
-                        self.log(self.msg)
+                        self.log(self.msg, "ERROR")
                         break
 
         except Exception as e:
-            error_message = "Error while Resyncing device in Cisco DNA Center - {0}".format(str(e))
-            self.log(error_message)
-            raise Exception(error_message)
+            self.status = "failed"
+            error_message = "Error while resyncing device in Cisco Catalyst Center: {0}".format(str(e))
```

Comment:
Which device? IP address or name ?

Reply:
Device resync api take list of devices at once, so I believe device ip should come as part of exception only.